src/shared/components/Settings/Account/MyAccount/index.jsx

```diff
@@ -101,7 +101,7 @@ export default class MyAccount extends React.Component {
   onUpdateNewEmailInput(e) {
     const newEmail = e.target.value;
     this.setState({
-      newEmail,
+      newEmail: newEmail.charAt(0).toUpperCase() + newEmail.slice(1),
```

Comment:
@ThomasKranitsas so is it allowed to type uppercase on desktop?

Reply:
It is allowed but we shouldn't enforce that.

By default, mobile devices will set the first letter to uppercase which looks weird.
That's what you need to disable

Reply:
@ThomasKranitsas u mean , user should not be allowed to enter input itself if it is uppercase?

Reply:
no no... A user can enter whatever case he/she wants.
The issue is that if I try to enter `email@example.com` from mobile, it will be entered as `Email@example.com`.

We should disable the default "Auto-capitalize" functionality.

Reply:
I think the fix would be to just add the `autoCapitalize="off"` attribute to the input field.

Reply:
@ThomasKranitsas 
1.so automatically keyboard shall be in small letter when user trying to enter first letter, 
2.rather keeping it capital in keyboard and then when user types it,but reflects in small,

Reply:
Please see my above comment. Just use the `autoCapitalize="off"` to fix this issue.

Reply:
@ThomasKranitsas thanks. i will immediatly update PR.just 1 min